web/app/js/components/ErrorModal.jsx

```diff
@@ -124,12 +124,27 @@ export default class ErrorModal extends React.Component {
     });
   }
 
+  renderIconStatus = errors => {
+    let containers = errors.byPodAndContainer;
+    let showInit = _.flatten(_.map(containers, container => {
```

Comment:
we're essentially creating showInit in order to pass it to the `_.every` below, but we might as well just check the `reason` in this loop, and avoid having two steps.

```
    let showInit = true;

    _.each(errors.byPodAndContainer, container => {
      _.each(container.byContainer, con => {
        if (con[0].reason !== "PodInitializing") {
          showInit = false;
        }
      });
    });

    return showInit ? <Badge status="processing" className="controller-init-icon" onClick={this.showModal} /> :
    <Icon type="warning" className="controller-error-icon" onClick={this.showModal} />;
```

Reply:
Yea, this is a great suggestion! I couldn't wrap my head around the different ways of traversing the error object but this is much cleaner and easier to read.